ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/DataItemListModel.h

```diff
@@ -18,12 +18,14 @@
 
 class DataItem;
 
+#include "Iterable.h"
+
 #include <QAbstractListModel>
 #include <QByteArray>
 #include <QHash>
 #include <QList>
 
-class DataItemListModel : public QAbstractListModel
+class DataItemListModel : public QAbstractListModel, public Esri::ArcGISRuntime::Iterable<DataItem*>
```

Comment:
Forgot I did this. Rather than use an ugly manual for loop in SampleManger.cpp i just inherited our iterable and updated some properties to enable ranged for loops for this ListModel and `SampleListModel` as well.